spec/controllers/v0/sessions_controller_spec.rb

```
@@ -472,10 +473,11 @@ def verify_session_cookie
                 errors: [
                   'Subject did not consent to attribute release',
                   'Other random error'
-                ]
+                ],
+                code: '007'
```

this shouldn't be 007 should it? I thought we had a unique message for this one too. Shouldn't this be 001?

---

I thought 007 for multiple errors.  Is that not right?

---

oh, this is what we do for multi-error.... HMMM.... yuck, that's a weird one to think about. I wonder if maybe we'd want to surface the error we know about. User resolves that issue, and then the next time they sign in, it will only be one error...

---

not even sure this one ever really happens to be honest...

---

I'm trying to keep this PR to the minimum, but it's certainly something to address in the refactor.